integration-test/1491-malaysian-road-shields.py

```diff
@@ -0,0 +1,509 @@
+            }),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 4937362,
+                'network': u'MY:federal',
+                'shield_text': u'3214',
+            })
+
+    def test_111b1_myfederal(self):
+        import dsl
+
+        z, x, y = (16, 52853, 32493)
+
+        self.generate_fixtures(
+            dsl.is_in('MY', z, x, y),
+            # https://www.openstreetmap.org/way/68058997
+            dsl.way(68058997, dsl.tile_diagonal(z, x, y), {
+                'access': u'yes',
+                'cycleway': u'no',
+                'highway': u'trunk',
+                'horse': u'no',
+                'int_ref': u'AH150',
+                'lanes': u'2',
+                'lit': u'yes',
+                'name': u'Jalan Datuk Amar Kalong Ningkan',
+                'oneway': u'yes',
+                'ref': u'1-11B1',
+                'source': u'openstreetmap.org',
+                'surface': u'asphalt',
+            }),
+            dsl.relation(1, {
+                'alt_name': u'Pan-Borneo Highway',
+                'int_ref': u'AH150',
+                'name': u'Asian Highway AH150',
+                'name:de': u'Asien Fernstraße AH150',
+                'name:en': u'Asian Highway AH150',
+                'name:fr': u'Route asiatique AH150',
+                'name:id': u'Jalan Asia AH150',
+                'name:ja': u'アジアハイウェイ150号線',
+                'name:ko': u'아시안 하이웨이 150호선',
+                'name:ms': u'Lebuhraya Asia AH150',
+                'name:ru': u'Азиатский маршрут AH150',
+                'name:vi': u'Đường Xuyên Á AH150',
+                'name:zh': u'亚洲公路150号线',
+                'network': u'AsianHighway',
+                'ref': u'AH150',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+                'wikipedia': u'en:Pan-Borneo Highway',
+            }, ways=[68058997]),
+            dsl.relation(2, {
+                'name': u'Laluan Persekutuan 1 (Sabah & Sarawak) (arah timur)',
+                'name:en': u'Federal Route 1 (Sabah & Sarawak) (eastbound)',
+                'name:zh': u'联邦路线1（沙巴和沙捞越）（东向）',
+                'network': u'my:federal',
+                'ref': u'1',
+                'role': u'east',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+            }, ways=[68058997]),
+        )
+
+        # TODO! should the shield text be '1' or '1-11B1'? not sure from
+        # available sources whether the '-11B1' part is important and/or
+        # usually on signs.
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 68058997,
+                'network': 'MY:federal',
+                'shield_text': '1',
+                'all_networks': ['MY:federal', 'AsianHighway'],
+                'all_shield_texts': ['1', 'AH150'],
```

Comment:
`AH150` > `150`?

Reply:
@nvkelso and @sensescape to research...

Reply:
From the Wikipedia page looks like we should really do AsianHighway:CountryCode to make the artwork functional per country:

- https://en.wikipedia.org/wiki/Template:AHN-AH

Reply:
Let's just file a followup issue for this and ignore it for this sprint.